### ISSUE TITLE
Weird result from Float64 addition

### ISSUE THREAD

**Issue by Absicola** [JuliaLang/julia#33867]
I have isolated a problem from one of my projects whereby I'm getting an unexpected result from adding 0.07 to 0.5. See code example below:
`num = 0.5
offset = 0.07
println(num+offset)`

Which outputs:
> 0.5700000000000001

I wondered if this was a precision issue, but assumed it unlikely to be that, considering a call to `typeof()` for both `num` and `offset` returned Float64. The result of the addition returns a type of Float64 too.

Is anyone able to identify the reason for this behaviour please? Adding 0.5 and 0.06 outputs 0.56 as I would expect.

I am using Julia Version 1.2.0 on macOS Mojave 10.14.6.

**Comment by KristofferC** [comment]
See for example https://discourse.julialang.org/t/psa-floating-point-arithmetic/8678.

TLDR: This is just how floats work.

**Comment by Absicola** [comment]
Thank you for the link, good to know for future. I will close the issue now.
